src/commands/init.js

```diff
@@ -24,11 +40,38 @@ function handleInitError(rawError) {
   }
 }
 
+async function handleDatabaseConfiguration() {
+  if (process.env.DATABASE_URL) {
+    return null;
+  }
```

Comment:
I think we want the success message "✅Checking your database setup" even when we figured out that there was already a DATABASE_URL set up.